UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -176,12 +176,20 @@ public void AuthChange(ulong steamid, ulong ownerid, ServerAuth.Status status)
 			return;
 		}
 
+		//god please work, should be the leave message and associated logging.
+		if (status == ServerAuth.Status.VACCheckTimedOut)
+		{
+
+			Logger.LogWarning($"The SteamID '{SteamID}' left the server. ({status})");
```

Comment:
Use of Logger requires a Category, otherwise it gets dumped into the Category.None pile and probably is never seen

Reply:
Added a reference to Category.Steam, please review.